recipe/meta.yaml

```diff
@@ -20,7 +19,6 @@ requirements:
   host:
     - pip
     - python {{ python_min }}
-    - setuptools
```

Comment:
Re-add it b/c you need a backend. Even thou this is passing now, it may fail in future Pythons.